csrankings-a.csv

```diff
@@ -1765,6 +1766,7 @@ Antoine Kaufmann [SWS],Max Planck Society,https://people.mpi-sws.org/~antoinek,6
 Antoine Limasset,CRIStAL,https://www.cristal.univ-lille.fr/profil/alimasse,EW4USK4AAAAJ
 Antoine Nongaillard,CRIStAL,https://www.cristal.univ-lille.fr/profil/anongail,NOSCHOLARPAGE
 Antoine Vigneron,UNIST,http://algo.unist.ac.kr,MIqVnpwAAAAJ
+Anton Burtsev,University of Utah,https://www.ics.uci.edu/~aburtsev,mkXzaOIAAAAJ

```

Comment:
No indication he is at Utah at the moment. Dead link on Utah faculty page. LinkedIn and home page says UC Irvine.